cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -134,6 +134,14 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 #}
+    {% block javascript_loader %}
```

Comment:
We should just recombine base-common.html with base.html.  Most of the components in base.html can be overridden.

Reply:
Yes! Though let's give that its own PR.